### PR TITLE
July community gems

### PR DESCRIPTION
**Pull request by flippedcoder**
These are the Community Gems for July. Just getting the Q&As reviewed earlier so we can schedule for it better. It still needs a few updates like the image and the August meetup.

### CODE REVIEW

File: content/blog/2021-07-27-july-21-dvc-community-gems.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+pick the VPC that manages that precise security group.
+
+We still don't provide a way of specifying VPCs other than the default one, but
+it's an issue that we're currently working on:
+https://github.com/iterative/terraform-provider-iterative/issues/107
+
+### [Q: Is it possible to rename and modify a file inside a directory tracked by DVC in one commit/change?](https://discord.com/channels/485586884165107732/485596304961962003/849589484517588992)
+
+If you modify the name and modify the file, you just need to run `dvc commit` or
+`dvc add` and then commit the change into Git.
```
Comment (dberenbaum) [inline content/blog/2021-07-27-july-21-dvc-community-gems.md:68]:
```suggestion
If you modify the name and modify the file, you just need to run `dvc commit` and then commit the change into Git.
```

Not sure why `dvc add` was mentioned in this answer, but it seems a bit irrelevant for a directory that's already tracked.

File: content/blog/2021-07-27-july-21-dvc-community-gems.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+second machine. Check your `.dvc/config` file for the second machine to make
+sure there aren't any errors. It could be something as simple as a connection
+string without the necessary quotation marks!
+
+Thanks so much for this question @raharth!
+
+### [Q: `dvc push` says, "Everything is up to date." However, I modified my dataset and this is confirmed with `dvc status`, where it lists a "modified" entry on the changed outs. How can I force a push of my changes?](https://discord.com/channels/485586884165107732/485596304961962003/857931383476977695)
+
+You need to run `dvc add` to create a `.dvc` file to track the new data. Then
+run `dvc commit` to commit your changes to the cache.
```
Comment (dberenbaum) [inline content/blog/2021-07-27-july-21-dvc-community-gems.md:101]:
```suggestion
You need to run `dvc commit` to commit your changes to the cache.
```

Again, it seems like this question is about data that's already tracked, so IMO `dvc add` is a bit misleading in the answer.

**Review by dberenbaum — APPROVED** [review]
Very minor comments. LGTM!

File: content/blog/2021-07-27-july-21-dvc-community-gems.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+Nice job working with the Python API @harry134!
+
+You can use the `Repo` API like this.
+
+```dvc
+from dvc.repo import Repo
+
+repo = Repo()
+repo.pull()
```
[the remark this answers is not in the document]
Comment (flippedcoder) [reply content/blog/2021-07-27-july-21-dvc-community-gems.md:115]: Good catch!

File: content/blog/2021-07-27-july-21-dvc-community-gems.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+This is a great question to help us all understand something so thanks
+@adwivedi.
+
+To look at your queued experiments, run `dvc exp show`. All of the queued
+experiments will be marked with an asterisk `*`.
+
+_Queued experiments are not shown with the `dvc exp list` command at the
+moment._
+
+### [Q: I have two machines and a central repo on Azure. With my second machine, I want to pull the dataset from the first machine. How can I pull the data with DVC?](https://discord.com/channels/485586884165107732/485596304961962003/859034882297823233)
```
[the remark this answers is not in the document]
Comment (flippedcoder) [reply content/blog/2021-07-27-july-21-dvc-community-gems.md:83]: I got rid of that Azure reference.

File: content/blog/2021-07-27-july-21-dvc-community-gems.md (hunk 1 above)
Comment (flippedcoder) [reply content/blog/2021-07-27-july-21-dvc-community-gems.md:68]: Another thing that's good to know. 🙂 So `dvc add` is how users add files to be track and `dvc commit` is how they add them to the DVC cache?

Comment (dberenbaum) [reply content/blog/2021-07-27-july-21-dvc-community-gems.md]: `dvc add` does both, but once they are being tracked, `dvc commit` saves changes to the cache.

File: content/blog/2021-07-27-july-21-dvc-community-gems.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+date: 2021-07-27
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: self-hosted runners, DVC commits,
+  troubleshooting remotes, and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: self-hosted runners, DVC commits,
+  troubleshooting remotes, and more.
+picture: 2021-07-27/gems-cover.png
```
Comment (shcheklein) [inline content/blog/2021-07-27-july-21-dvc-community-gems.md:12]: image is not being rendered I think ... are we waiting on it still?

File: content/blog/2021-07-27-july-21-dvc-community-gems.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/july-21-community-gems/779
+tags:
+  - Community
+  - Runners
+  - Remotes
+  - CML
+  - DVC
+  - Gems
```
Comment (shcheklein) [inline content/blog/2021-07-27-july-21-dvc-community-gems.md:21]: Minor: so, do we use this or not :) I though that we decided that the one example we saw was made by mistake and we should fix that one.

File: content/blog/2021-07-27-july-21-dvc-community-gems.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+  A roundup of technical Q&A's from the DVC community.
+  This month: self-hosted runners, DVC commits,
+  troubleshooting remotes, and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: self-hosted runners, DVC commits,
+  troubleshooting remotes, and more.
+picture: 2021-07-27/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/july-21-community-gems/779
```
Comment (shcheklein) [inline content/blog/2021-07-27-july-21-dvc-community-gems.md:14]: reminder: don't forget to fix the link before merge

File: content/blog/2021-07-27-july-21-dvc-community-gems.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+Good question @BSVogler.
+
+### [Q: I'm trying to use the DVC API in a Jupyter notebook. Can I simulate a `dvc push` command via the API?](https://discord.com/channels/485586884165107732/485596304961962003/856979475068878898)
+
+Nice job working with the Python API @harry134!
+
+You can use the `Repo` API like this.
+
+```dvc
```
Comment (shcheklein) [inline content/blog/2021-07-27-july-21-dvc-community-gems.md:110]: dvc -> python

**Review by shcheklein — APPROVED** [review]
All looks good to me, let's not forget to fix minor things and placeholders that we have in the text.

File: content/blog/2021-07-27-july-21-dvc-community-gems.md (hunk 5 above)
Comment (flippedcoder) [reply content/blog/2021-07-27-july-21-dvc-community-gems.md:12]: Yep! We're still waiting on this.

File: content/blog/2021-07-27-july-21-dvc-community-gems.md (hunk 6 above)
Comment (flippedcoder) [reply content/blog/2021-07-27-july-21-dvc-community-gems.md:21]: Yeah, we did decide that. 😅  I deleted the `Gems` tag from this and the other one to keep things consistent.

File: content/blog/2021-07-27-july-21-dvc-community-gems.md (hunk 5 above)
Comment (jorgeorpinel) [reply content/blog/2021-07-27-july-21-dvc-community-gems.md:12]: Is this the only thing missing? Do you guys have the ability to merge? @flippedcoder @jendefig otherwise please tag us when ready 🙂

Comment (jendefig) [reply content/blog/2021-07-27-july-21-dvc-community-gems.md:12]: @flippedcoder Are you going to put the pic in or do you need me to?
